Task2/stack.c

```diff
@@ -0,0 +1,104 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include "stack.h"
+
```

Comment:
```suggestion

typedef struct StackElement
{
    int value;
    struct StackElement* nextElementPtr;
} StackElement;
typedef struct Stack
{
    StackElement* lastElementPtr;
} Stack;

```
Вот тут должен быть описан тип